config/initializers/lograge.rb

```diff
@@ -1,11 +0,0 @@
-HarvesterWorker::Application.configure do
```

Comment:
check with Dan if we are using lograge

Reply:
We are going to use the default logging from Rails 5 now and see how it goes.